bin/generate-ssl-options.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+#
+# Script to generate private RSA key and a self-signed certificate under src/main/resources/localhost-ssl (git-ignored)
+# These resources are used for exposing the application via HTTPS in development environment. They must not be used
+# in other environments, as their purpose is to provide consistency, not security.
+
+localhost_ssh_folder="src/main/resources/localhost-ssl"
+
+if [ -f "$localhost_ssh_folder"/localhost.key -a -f "$localhost_ssh_folder"/localhost.crt ]
+then
+  echo "SSL files exist already. Going to re-use them"
```

Comment:
I would invert the if statement, and not log this, it seems like a statement that isn't valuable and just pollutes startup logs, not a blocker